testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/http/SpnegoMechTestCase.java

```diff
@@ -174,49 +163,33 @@ public void testSuccess() throws Exception {
     /**
      * A setup task which configures and starts Kerberos KDC server.
      */
-    @CreateDS(
-            name = "JBossDS-SpnegoMechTestCase",
-            factory = org.jboss.as.test.integration.ldap.InMemoryDirectoryServiceFactory.class,
-            partitions = {
-                    @CreatePartition(
-                            name = "wildfly",
-                            suffix = "dc=wildfly,dc=org",
-                            contextEntry = @ContextEntry(
-                                    entryLdif = "dn: dc=wildfly,dc=org\n" +
-                                                "dc: wildfly\n" +
-                                                "objectClass: top\n" +
-                                                "objectClass: domain\n\n"))
-            },
-            additionalInterceptors = { KeyDerivationInterceptor.class })
-    @CreateKdcServer(
-            primaryRealm = "WILDFLY.ORG",
-            kdcPrincipal = "krbtgt/WILDFLY.ORG@WILDFLY.ORG",
-            searchBaseDn = "dc=wildfly,dc=org",
-            transports = {
-                    @CreateTransport(protocol = "UDP", port = 6088)
-            })
     static class KDCServerSetupTask implements ServerSetupTask {
 
-        private DirectoryService directoryService;
-        private KdcServer kdcServer;
+        private SimpleKdcServer simpleKdcServer;
 
         @Override
         public void setup(ManagementClient managementClient, String containerId) throws Exception {
-            directoryService = DSAnnotationProcessor.getDirectoryService();
-            final SchemaManager schemaManager = directoryService.getSchemaManager();
-            for (LdifEntry ldifEntry : new LdifReader(SpnegoMechTestCase.class.getResourceAsStream(NAME + ".ldif"))) {
-                directoryService.getAdminSession().add(new DefaultEntry(schemaManager, ldifEntry.getEntry()));
-            }
-            kdcServer = KDCServerAnnotationProcessor.getKdcServer(directoryService, 1024, "localhost");
+
+            simpleKdcServer = new SimpleKdcServer();
+            simpleKdcServer.setKdcHost("localhost");
+            simpleKdcServer.setKdcRealm("WILDFLY.ORG");
+            simpleKdcServer.setAllowUdp(true);
+            simpleKdcServer.setKdcUdpPort(6088);
+            simpleKdcServer.setInnerKdcImpl(new DefaultInternalKdcServerImpl(simpleKdcServer.getKdcSetting()));
+
+            simpleKdcServer.init();
+
+            simpleKdcServer.createPrincipal("HTTP/localhost@WILDFLY.ORG","httppwd");
+            simpleKdcServer.createPrincipal("user1@WILDFLY.ORG", "password1");
```

Comment:
Just minor, since "user1@WILDFLY.ORG" is used elsewhere in this class, would be good to make use of a constant here too. Same for the password.